packages/styles/scss/components/masthead/_masthead-l1.scss

```diff
@@ -12,6 +12,14 @@
 $search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
 $search-transition-timing: 95ms;
 
+@mixin mastheadL1Short {
+  height: 48px;
```

Comment:
Maybe use `carbon--mini-units(6)` instead of 48px

Reply:
48px is actually how it is right now with the media query, but I could implement mini-units, I guess.

Reply:
Done, thanks for the suggestion!